go.mod

```diff
@@ -4,6 +4,10 @@ go 1.18
 
 require (
 	github.com/alecthomas/chroma v0.10.0
+	github.com/aws/aws-sdk-go-v2 v1.17.3
+	github.com/aws/aws-sdk-go-v2/config v1.18.7
+	github.com/aws/aws-sdk-go-v2/feature/s3/manager v1.11.46
+	github.com/aws/aws-sdk-go-v2/service/s3 v1.29.6
```

Comment:
Do you suppose you could collect little before-and-after numbers for what this does to the compiled binary sizes?

Features probably win over worrying about binary size -- but I'd like to be informed about the tradeoffs we're making.

(And yes, maybe feed this statistic forward into decisions about if we want to isolate these features into other binaries.  Whether today or in the future.)

Reply:
You can somewhat get this number from the CI build artifact. If you want something more detailed, it should probably emit from the build CI job.

master | 5f3e22b7e86cf53be904bf36b8e51d4478fdf065: 23.7MB
branch | 93ec2c1a24cc3555eed1bfac22564ad911c51716:  26.4 MB

Reply:
I think < 3 MB is sufficiently small that we don't care?

Reply:
FYI that's tarball size. I didn't decompress it.

Reply:
Binary from `master` is 28M
Binary from this branch is 35M

Reply:
Yeah, we don't love that this adds to bin size, but for now we deemed this to be okay. Also note, we can save more than this adds with build flags to remove debug info!